src/Command/TaskCommand.php

```diff
@@ -49,6 +49,6 @@ public function fileName(string $name): string
      */
     public function template(): string
     {
-        return 'Shell/task';
+        return 'Bake.Shell/task';
```

Comment:
This seems fine to me. Plugins offering bake commands already need to do something similar.

Reply:
The goal is to make Bake use config and conventions instead of legacy hacks. This helps ensure we're supporting it properly.